indico/modules/events/editing/operations.py

```diff
@@ -247,16 +247,31 @@ def undo_review(revision):
 
 
 @no_autoflush
-def update_review_comment(revision, text):
+def update_review(revision, text, tags):
```

Comment:
I'd take a `data` dict here, then you can use `@use_rh_args(...)` in the PR and just pass that (and even ditch the `load_default`)